packages/config/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@redhat-cloud-services/frontend-components-config",
-    "version": "4.0.8",
+    "version": "4.0.9-beta.1",
```

Comment:
No need to push this, release bot will override it.

Reply:
The bot will make 4.0.9 by itself. It did that before.